sdk/examples/wallet/storage.rs

```diff
@@ -1,11 +1,12 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! cargo run --example storage --release
+//! TODO: Example description
```

Comment:
TODO

Reply:
Not really part of this PR. Even making this stuff more consistent can be argued not part of this PR since it's basically just about a dep change.I'll create an issue for that so that someone (best would be the author of the example) can fix the todo.